src/i18n/messages/all.en.json

```diff
@@ -206,10 +206,20 @@
     "i18nStrings.operatorText": "Operator",
     "i18nStrings.operatorsText": "Operators",
     "i18nStrings.propertyText": "Property",
+    "i18nStrings.removeTokenButtonAriaLabel": "Remove filter, {token__formattedText}",
```

Comment:
Note: the change to removeTokenButtonAriaLabel and the new labels are only provided for all.en locale now. That is expected: the old and new format of the removeTokenButtonAriaLabel are consistent and the new labels are only used in the internal permutations as of yet.